packages/cactus-plugin-keychain-aws-sm/README.md

```diff
@@ -0,0 +1,3 @@
+# `@hyperledger/cactus-plugin-keychain-aws-sm`
```

Comment:
@Leeyoungone Is there some other PR accidentally mixed into this one? This file is being added by another PR by Jagpreet IIRC.

Reply:
oof, I think it is accidentally mixed into that one. I'll change it soon. Thank you for catching it!!

Reply:
@Leeyoungone No worries at all, once ready for review, please hit the re-request review button and I'll come back around to it.